### ISSUE TITLE
infracost plugin authorization issues

### ISSUE THREAD

**Issue by jsidney** [veecode-platform/platform-backstage-plugins#32]
Hey guys :) my team has just installed and configured the infracost plugin. but for some reason, whenever i try to navigate to the infracost tab, i get an error 400 and the logs tell me that:
```json
{
  "error": {
    "name": "AuthenticationError",
    "message": "Missing credentials"
  },
  "request": {
    "method": "GET",
    "url": "/api/infracost/platform-infrastructure"
  },
  "response": {
    "statusCode": 401
  }
}
```

i have added the following permission block into my policy:
```js
...
import { infracostCreatePermission, infracostDeletePermission, infracostReadPermission, infracostUpdatePermission } from '@veecode-platform/backstage-plugin-infracost-common';
...
const infracostPermissions = [
  infracostCreatePermission.name,
  infracostDeletePermission.name,
  infracostReadPermission.name,
  infracostUpdatePermission.name
]
...
 if (infracostPermissions.includes(request.permission.name)) {
      return { result: AuthorizeResult.ALLOW };
  }
```

has anyone else had similar issues and do you know how solve it :) ?

**Comment by jsidney** [comment]
from what i can see, the authorization header is not being passed along with the api request from the frontend and the backend.

**Comment by jsidney** [comment]
i am not entirely sure what the bug really is. but it doesnt seem to be releated to the auth or permissions.

**Comment by ValberJunior** [comment]
Hi, thanks for the feedback. Can you let me know which version of the plugin you are using and if you are already using the new backend ?

**Comment by ValberJunior** [comment]
This bug was identified using the new backend, but has now been resolved as of version 0.3.8.
